src/database/migrations/20191025142731-create-user.js

```diff
@@ -32,12 +32,12 @@ module.exports = {
       createdAt: {
         allowNull: false,
         type: Sequelize.DATE,
-        defaultValue: Sequelize.NOW
+        defaultValue: Sequelize.NOW,
```

Comment:
Lets make this uniform like the others like so:
field: 'created_at',

Reply:
Ok,will look into that,

Reply:
please use the field attribute here like @iphyokafor just said

Reply:
okay